src/com/lftechnology/java/training/alina/basic/dayname/CommonConstants.java

```diff
@@ -7,16 +7,16 @@
  *
  */
 public class CommonConstants {
-	public static final String SUNDAY = "Sunday";
-	public static final String MONDAY = "Monday";
-	public static final String TUESDAY = "Tuesday";
-	public static final String WEDNESDAY = "Wednesday";
-	public static final String THURSDAY = "Thursday";
-	public static final String FRIDAY = "Friday";
-	public static final String SATURDAY = "Saturday";
-	public static final String INVALID_DAY = "Not a valid day";
-	
-	private CommonConstants() {
-		// TODO Auto-generated constructor stub
-	}
+    public static final String SUNDAY = "Sunday";
+    public static final String MONDAY = "Monday";
+    public static final String TUESDAY = "Tuesday";
+    public static final String WEDNESDAY = "Wednesday";
+    public static final String THURSDAY = "Thursday";
+    public static final String FRIDAY = "Friday";
+    public static final String SATURDAY = "Saturday";
+    public static final String INVALID_DAY = "Not a valid day";
+
+    private CommonConstants() {
+        // TODO Auto-generated constructor stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)